NUnitLite/TouchRunner/TcpTextWriter.cs

```diff
@@ -17,24 +18,42 @@ namespace MonoTouch.NUnit {
 	public class TcpTextWriter : TextWriter {
 		
 		private TcpClient client;
+		TcpListener server;
 		private StreamWriter writer;
 
-		public TcpTextWriter (string hostName, int port)
+		public TcpTextWriter (string hostName, int port, bool isTunnel = true)
```

Comment:
hmm... people updating will be hit by the change - unless the touch.server code is kept in sync
https://github.com/spouliot/Touch.Unit/tree/master/Touch.Server
That's how other people are using the automation (directly or based on that code) from old blog posts.

Reply:
ok, I'll address that.